### PR TITLE
format docstrings, second round same as the first

### PR DESCRIPTION
**Pull request by bramtayl**
Rebased

### CODE REVIEW

**Comment by bramtayl** [comment]
The bug was just deleting necessary initial whitespace so that should be fixed

**Comment by bramtayl** [comment]
Whack-a-bug

File: test/default_style.jl [hunk 1]
```diff
@@ -4836,4 +4844,119 @@ some_function(
...
+        formatted = """
+        \"""
+        ```
+        format_text(
+        )
+        ```
+        \"""
+        function format_text() end"""
+
+        @test format_text(unformatted) == formatted
```
Comment (domluna) [inline test/default_style.jl:4945]: What's the rationale for reformatting like this?

**Comment by domluna** [comment]
@bramtayl 

So I think i fixed the bug - https://github.com/domluna/JuliaFormatter.jl/commit/c6150285b1ca4adca74ba470d0170121b7cc3135

Also I've been playing around with CommonMark, it looks promising but I'm not sure it's suitable as of yet.

If this is merged it'll be marked as experimental since it might eat some parts of docstrings, as shown in the test that had to be labelled as broken.

File: test/default_style.jl (hunk 1 above)
Comment (bramtayl) [reply test/default_style.jl:4945]: I don't know. `Markdown.plain` does it though

**Comment by bramtayl** [comment]
Yay!!! I'm looking forward to very pretty docstrings

**Comment by bramtayl** [comment]
Close in favor of #254

**Pull request closed without merging**